### PR TITLE
add auto_update_docker_image

### PR DESCRIPTION
**Pull request by JudahSchwartz**
## Related Issues
fixes:

### CODE REVIEW

**Review by GuyAfik — COMMENTED** [review]
@JudahSchwartz Add a changelog

File: demisto_sdk/commands/content_graph/tests/parsers_and_models_test.py [hunk 1]
```diff
@@ -1263,6 +1263,7 @@ def test_script_parser(self, pack: Pack):
         )
         assert model.type == "python"
         assert model.subtype == "python3"
+        assert model.auto_update_docker_image
```
Comment (GuyAfik) [inline demisto_sdk/commands/content_graph/tests/parsers_and_models_test.py:1266]: maybe add a test to make sure its populated with false when it exists?

File: demisto_sdk/commands/content_graph/parsers/integration_script.py [hunk 2]
```diff
@@ -86,3 +86,9 @@ def subtype(self):
         if not subtype and self.type == "python":
             subtype = "python2"
         return subtype
+
+    @property
+    def auto_update_docker_image(self):
+        return (
+            get_value(self.yml_data, self.field_mapping.get("type", "")) or ""
```
[the remark this answers is not in the document]
Comment (JudahSchwartz) [reply demisto_sdk/commands/content_graph/parsers/integration_script.py:93]: forgot to overwrite. fixing

File: .changelog/4133.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+changes:
+- description: Added `auto_update_docker_image` to IntegrationScript graph object.
```
Comment (GuyAfik) [inline .changelog/4133.yml:2]:
```suggestion
- description: Added the `auto_update_docker_image` field to the content-graph for scripts and integrations.
```

**Review by GuyAfik — APPROVED** [review]
⚡

**Pull request merged**